src/main/resources/application.yaml

```diff
@@ -8,6 +8,12 @@ management:
 s2s:
   url: ${S2S_URL}
 
+idam:
+  s2s-auth:
+    url: ${S2S_URL}
+    totp_secret: ${TOTP_SECRET}
+    microservice: jobscheduler
```

Comment:
Not sure about the current convention (worth checking ansible scripts), it may use dashes.